### ISSUE TITLE
System Error

### ISSUE THREAD

**Issue by DarlingHang** [taldatech/soft-intro-vae-pytorch#10]
Hello, I am training soft-intro-vae with my own dataset. And these lines of codes always raise a SystemError:
if torch.isnan(lossD) or torch.isnan(lossE): raise SystemError
Can you explain this?

**Comment by taldatech** [comment]
Yes, you loss diverges and results in NaN. I'd need more information to help you, e.g., resolution, total data samples, and the hyper-parameters you used.

For now, I'd suggest trying setting `beta_neg` to a higher value, to reduce the repulsion power of fake data in the encoder.

**Comment by taldatech** [comment]
I'm closing the issue for now, re-open it if you need further assistance.
